pkgs/development/ocaml-modules/terml/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildDunePackage, fetchFromGitHub, lib, uutf }:
+
+buildDunePackage rec {
+  pname = "terml";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "wllfaria";
+    repo = "terml";
+    rev = "${version}";
+    hash = "sha256-2ifMfUaYYsCFOACgXgJ5IuoSEicHyIqumLpun2ZqcDc=";
+  };
+
+  propagatedBuildInputs = [ uutf ];
+
+  duneVersion = "3";
+
```

Comment:
```suggestion
  minimalOCamlVersion = "4.13";

```

Reply:
Fine by me. I force-pushed the change to my fork.